stdlib/array.gr

```diff
@@ -70,7 +70,7 @@ provide let length = array => {
  * @since v0.1.0
  */
 @unsafe
-provide let make: (Number, a) -> Array<a> = (length: Number, item: a) => {
+provide let make = (length: Number, item: a) => {
```

Comment:
Why'd you need to delete these? Should we instead change them to the labeled signature?

Reply:
Related to #1775; I could change them to the labeled signature but I figured it would just be extraneous information since the param types and return type are already specified in the implementation